hphp/runtime/vm/jit/vasm-lower.cpp

```diff
@@ -195,12 +195,12 @@ void lower_vcall(Vunit& unit, Inst& inst, Vlabel b, size_t i) {
             v << copy2{rret(0), rret(1), dests[0], dests[1]};
             break;
           case Arch::ARM:
-            // For ARM64 we are truncating the aux data
-            // from the type. That allows to use the resulting
-            // register values to be used in type comparisons
-            // without the need for truncation there.
+            // For ARM64 we need to clear the bits 8..31 from the type value.
+            // That allows to use the resulting register values to be used
```

Comment:
`s/That allows to use the resulting register values to be used/That allows us to use the resulting register values/`, and reflow—here and below.

Can you also mention that we leave bits 32..63 alone in order to preserve the m_aux member?